samples/bluetooth/st_ble_sensor/README.rst

```diff
@@ -0,0 +1,59 @@
+.. _bluetooth-st_ble_sensor:
+
+Bluetooth: ST BLE Sensor Demo
+#############################
+
+Overview
+========
+This application demonstrates BLE peripheral by exposing vendor-specific
+GATT services.Currently only button notification and LED service are
+implemented.Other BLE sensor services can easily be added.
+Check `BlueST protocol`_ document.
+Although this sample can work with non stm32wb55 hardware it was tested
```

Comment:
This sentence is confusing, maybe because I'm unfamiliar with the hardware names you're using.  I think you're saying you tested with a different board than listed in the example below (nucleo_wb55rg).  Can you clarify?

Reply:
I decided to develop the demo because the original ST program for use with ST BLE Demo is quite complex.Then I found that the ST BLE demo android app  works with hardware from another manufacturer.

Reply:
I decided to develop the demo because the original ST program for use with ST BLE Demo is quite complex.Then I found that the ST BLE demo android app works with hardware from another manufacturer.I believe ST can block access to other hardware in the future